test/e2e/sushy-tools/sushy-emulator.conf

```diff
@@ -0,0 +1,36 @@
+# Listen on all local IP interfaces
+SUSHY_EMULATOR_LISTEN_IP = u'0.0.0.0'
```

Comment:
This is not great. You use neither authentication nor TLS. This will open VM management on your machine to anyone. Cannot we assume localhost? If not, I'd probably add TLS and auth.

Reply:
Good point! I think we can use `192.168.222.1` here. Localhost will probably not work since Ironic would not be able to reach it then.